### DISCUSSION TITLE
[show-and-tell] a solution to getting greta to work with targets

### DISCUSSION THREAD

**Discussion by njtierney** [ropensci/targets#1185] [Show and tell]
### Help

- [X] I understand and agree to <https://books.ropensci.org/targets/help.html>.

### Description

Hi folks!

Just wanted to share a solution we made for getting the [greta](https://github.com/greta-dev/greta) package to work with targets. We were able to use a feature of greta that allows for reinitialising tensorflow objects, which means that we can now make them work within a targets workflow. 

I just thought I'd show off our current solution, which is to use a hook - https://github.com/idem-lab/example-greta-targets - in the future we will work on getting greta to resolve dead pointers automatically. But just thought it might be worthwhile sharing 😄 and getting feedback

**Comment by wlandau** [comment]
Awesome work @njtierney, @smwindecker, and @goldingn! The use of a hook to awaken models is fantastic!
